SpatialGDK/Source/SpatialGDKEditorToolbar/Private/SpatialGDKEditorToolbar.cpp

```diff
@@ -99,7 +100,7 @@ void FSpatialGDKEditorToolbarModule::StartupModule()
 
 	FEditorDelegates::PreBeginPIE.AddLambda([this](bool bIsSimulatingInEditor)
 	{
-		if (GIsAutomationTesting && GetDefault<UGeneralProjectSettings>()->UsesSpatialNetworking())
+		if (GIsAutomationTesting && USpatialStatics::IsSpatialNetworkingEnabled())
```

Comment:
The `SpatialStatics` version only exists for blueprint access, we can continue using the `GetDefault<USpatialGDKSettings>()->bSpatialNetworking;` style and avoid having an additional include.